.kokoro/docker/Dockerfile

```diff
@@ -174,4 +174,14 @@ RUN useradd -d /h -u ${UID} ${USERNAME}
 # Allow nopasswd sudo
 RUN echo "${USERNAME} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
+# Setup DRIFT region tag parser
+ENV REGION_TAG_PARSER_DIR /region-tag-parser
+ENV PARSER_PATH ${REGION_TAG_PARSER_DIR}/wizard-py/cli.py
+
+RUN git clone https://github.com/GoogleCloudPlatform/repo-automation-playground "$REGION_TAG_PARSER_DIR" --single-branch
+
+# Add commit hash so Docker rebuild images whenever region tag parser changes
+RUN cd ${REGION_TAG_PARSER_DIR} && git checkout 3a1a1c51ff92264d36489d8e9ceb1a9f2f760f6b
+RUN chmod +x $PARSER_PATH
+
```

Comment:
This certainly works.

However, if we change the commit hash, all the builds will be full builds.

Also, do you need this feature for all of the builds?

For example, is it ok to only run this in py-3.7 periodic build?

Sorry for being back and forth, but I'd like to avoid unnecessary full builds if possible.